pkg/descheduler/controllers/migration/controller.go

```diff
@@ -421,18 +437,11 @@ func (r *Reconciler) preparePendingJob(ctx context.Context, job *sev1alpha1.PodM
 	}
 
 	markPodPrepareMigrating(pod)
-	if !evictionsutil.HaveEvictAnnotation(job) {
-		if aborted, err := r.abortJobIfNonRetryablePodFilterFailed(ctx, pod, job); aborted || err != nil {
-			if err == nil {
-				err = fmt.Errorf("abort job since failed to non-retryable Pod filter")
-			}
-			return reconcile.Result{}, err
-		}
-		if requeue, err := r.requeueJobIfRetryablePodFilterFailed(ctx, pod, job); requeue || err != nil {
-			return reconcile.Result{RequeueAfter: defaultRequeueAfter}, err
-		}
-	}
 
+	// delete passed arbitration annotation
+	if job.Annotations != nil {
+		delete(job.Annotations, arbitrator.AnnotationPassedArbitration)
```

Comment:
Why delete the annotation here?

Reply:
`AnnotationPassedArbitration` is for `pending` or empty phase. Here, the PMJs' phases will turn to `running`.